Gordon360/Controllers/ScheduleController.cs

```diff
@@ -84,21 +84,21 @@ public async Task<ActionResult<JArray>> GetAsync(string username)
 
                 if (viewerGroups.Contains(AuthGroup.Police) || viewerGroups.Contains(AuthGroup.SiteAdmin))
                 {
-                    scheduleResult = await _scheduleService.GetScheduleStudentAsync(username);
+                    scheduleResult = await _scheduleService.GetScheduleStudentAsync(username, sessionID);
+
                 }
-                else if (viewerGroups.Contains(AuthGroup.Student) && schedulePrivacy == 0)
+                else if (viewerGroups.Contains(AuthGroup.Student) && schedulePrivacy == 1)
```

Comment:
Why was this changed? Was the previous version incorrect?

Reply:
@andrew-wzj was this residual from testing? This if statement makes it seem like: `if you're a student, and the person is private, you can see their items?` This seems like something you did in testing to see someone else and should be reverted. However, I could be mistaken so please clarify/correct me

Reply:
While testing the api, we found out that students could not view their own official course schedule because the logic in the api code didn't have a case for viewing your own schedule for students. @amos-cha and @jsenning helped us change the logic for users being able to see their own schedule by adding an extra if statement at the end for checking if the username matches the authenticatedUserUsername.